src/analysis/retail/priest/shadow/modules/spells/Voidbolt.tsx

```diff
@@ -53,9 +77,73 @@ class Voidbolt extends ExecuteHelper {
     });
   }
 
+  onVBUpdate(event: UpdateSpellUsableEvent) {
+    //this adds timestamps of voidbolt spellusable updates
+    this.VB.push(event.timestamp);
+  }
+
+  onVBCast() {
+    this.castVB += 1;
+  }
+
+  //VB is an unusal spell. It is likely that using ExecuteHelper would be better than this for most spells.
+  getVB() {
```

Comment:
would suggest a clearer title for this. `getX` methods usually just return `X`, but this builds the values of `miss`. maybe `calculateMissedVBs`?

Reply:
I have changed the name to make it clearer.